spec/services/atmosphere/billing_service_spec.rb

```diff
@@ -126,6 +148,10 @@
     let!(:alien_appl) { create(:appliance, fund: alien_fund, appliance_set: wf, appliance_type: not_shareable_appl_type, appliance_configuration_instance: config_inst, virtual_machines:[])}
 
     it 'can or cannot afford flavors' do
```

Comment:
It seems like this spec and the one below tests to much. Please try to test only one thing in one spec - it will allow other developers to understand behaviour easily.